OWNERS

```diff
@@ -1,16 +1,72 @@
-approvers:
-- enxebre
-- csrwng
-- sjenning
-- davidvossel
-- muraee
-- bryan-cox
-- jparrill
-options: {}
-reviewers:
-- enxebre
-- csrwng
-- sjenning
-- hasueki
-- rtheis
+filters:
+  ".*":
+    approvers:
+    - core-approvers
+    reviewers:
+    - core-reviewers
+    emeritus-approvers:
+    - davidvossel
```

Comment:
David is not in Kubevirt anymore, so you will need to add Oren Cohen for that matters.